src/components/shared/title/title.component.jsx

```diff
@@ -2,14 +2,37 @@ import React from 'react'
 
 import useTitleStyles from './title.styles'
 
-function Title ({ children }) {
+function Title ({ children, level = 2 }) {
```

Comment:
I would create an enum for the level, also I think that's better to name it `type` which can be `h1, h2, h3` instead of `level` and just a number `1, 2, 3

Reply:
Actually I added if there is any need to use it in the future. Now all headlines are h2. I've changed the `level` for `type` and also now they are named `h1`, `h2` etc.
Not sure what exactly do you mean by creating an enum for it? I don't see much space to simplify how it is now, but let me know.

Reply:
Yep, adding the type is enough, no need for the enum